meson.build

```diff
@@ -269,6 +269,12 @@ if get_option('plugin_thunderbolt')
   conf.set('HAVE_THUNDERBOLT', '1')
 endif
 
+if get_option('plugin_flashrom')
+  flashrom = find_program('flashrom', required: false)
+  libflashrom = cc.find_library('libflashrom', required: false)
+  conf.set('HAVE_FLASHROM', '1')
```

Comment:
I don't think we need this if the entire plugin is conditional on libflashrom existing.